workshops/models.py

```diff
@@ -159,6 +159,35 @@ def self_organized_workshops_per_year_remaining(self):
 #------------------------------------------------------------
 
 
+class Sponsorship(models.Model):
+    '''Represent sponsorship from a host for an event.'''
+
+    SPONSORSHIP_SLABS = (
```

Comment:
For me slab is [this](http://peppardbuildingsupplies.com/sites/default/files/SWATCH_BS_Grey_0.jpg) :-)

Anyway, what does `Sponsorship.slab` represent here?